src/sentry/static/sentry/app/types/utils.tsx

```diff
@@ -0,0 +1,8 @@
+// from:
+// - https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#assertion-functions
+// - https://www.typescriptlang.org/play/#example/assertion-functions
+
+// This declares a function which asserts that the expression called
+// value is true:
+// eslint-disable-next-line prettier/prettier
+export function assert(_value: unknown): asserts _value {}
```

Comment:
Shouldn't this have an assertion inside if the value isn't truthy?

Reply:
Nah. It shouldn't. It's not a runtime check.

Reply:
To clarify, this is a type assertion check. 

It's not to be confused with other utility functions like `invariant`, which is a runtime check: https://github.com/zertosh/invariant